From 0eef6194fcbc0ed68bd1164df1372e45e9d2e579 Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Wed, 10 Apr 2019 21:39:24 +0200
Subject: [PATCH] TRITON-1384 CNAPI/DAPI blows assertion "callback (Function)
 is required" when doing getVm Reviewed by: Trent Mick <trentm@gmail.com>
 Approved by: Trent Mick <trentm@gmail.com>

---
 package.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package.json b/package.json
index a6ef18a..9a4cd66 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "Triton Compute Node API",
-  "version": "1.24.0",
+  "version": "1.24.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -10,7 +10,7 @@
     "async": "1.5.2",
     "backoff": "2.5.0",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#f2b4e8dac18b400a64fedb27db869b8d32ef009b",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#066e52618ed6f4ef25121716b95b19cb8f90a5b4",
     "deep-object-diff": "1.1.0",
     "dox": "0.9.0",
     "gc-stats": "1.2.0",
-- 
2.21.0

